indy_node/test/attrib_txn/test_nym_attrib.py

```diff
@@ -251,3 +276,68 @@ def testTrustAnchorAddedAttributeCanBeChanged(sdk_added_raw_attribute):
     # TODO but only by user(if user has taken control of his identity) and
     # trustAnchor
     raise NotImplementedError
+
+
+def test_auth_rule_for_attrib_works(looper,
```

Comment:
This test is for `raw` attribute. It would be great to have similar tests for `hash` and `enc` attributes as well.

Reply:
ok